staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -74,6 +74,13 @@ const (
 type KubeletConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
 
+	// kubeConfig is the path to a kubeconfig file, specifying how to connect to the API server.
+	// Providing kubeconfig enables API server mode, omitting kubeconfig enables standalone mode.
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// the path of kubeconfig file is different with before, and this may disrupt your node.
+	// Default: ""
+	// +optional
+	KubeConfig string `json:"kubeConfig,omitempty"`
```

Comment:
We might consider lowercasing the `c` - since the standard usage is all-lowercase `kubeconfig`.

Reply:
if use ```kubeconfig```, will KUBE API rule check failed, so need to add a new API rule violation in ```api/api-rules/violation_exceptions.list```,  are you sure? @mtaufen

Reply:
To confirm, the following fails the test?
```
Kubeconfig string `json:"kubeconfig,omitempty"
```

Reply:
I mean in this form: 
```
KubeConfig string `json:"kubeconfig,omitempty"`
```

ok, i will change to the following format:
```
Kubeconfig string `json:"kubeconfig,omitempty"`
```